core/app/models/spree/store.rb

```diff
@@ -27,6 +27,12 @@ def self.default
       end
     end
 
+    def supported_currencies
```

Comment:
@btolarz maybe it would be better to make this another method like `supported_currencies_list` and not hack the `supported_currencies` attribute

Reply:
@btolarz let's also add presence validation for `supported_currencies` field